api_docs/drone_doc.py

```diff
@@ -0,0 +1,346 @@
+"""Generated API Documentation for Drone API using drone_doc_gen.py."""
+
+drone_doc = {
+    "@context": {
+        "ApiDocumentation": "hydra:ApiDocumentation",
+        "code": "hydra:statusCode",
+        "domain": {
+            "@id": "rdfs:domain",
+            "@type": "@id"
+        },
+        "expects": {
+            "@id": "hydra:expects",
+            "@type": "@id"
+        },
+        "hydra": "http://www.w3.org/ns/hydra/core#",
+        "label": "rdfs:label",
+        "property": {
+            "@id": "hydra:property",
+            "@type": "@id"
+        },
+        "range": {
+            "@id": "rdfs:range",
+            "@type": "@id"
+        },
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "readonly": "hydra:readonly",
+        "supportedClass": "hydra:supportedClass",
+        "supportedOperation": "hydra:supportedOperation",
+        "supportedProperty": "hydra:supportedProperty",
+        "title": "hydra:title",
+        "vocab": "http://hydrus.com/droneapi/vocab#"
+    },
```

Comment:
Some of the items are not mapped to IRIs, rest everything is fine.
![sketch 15](https://user-images.githubusercontent.com/9114274/28236543-1719afc8-6946-11e7-9523-02e6382afc67.png)

@chrizandr you should add these to the contexts as well. For example `description` should be mapped to `hydra:description`, the same goes for the server doc as well.